### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by gdestuynder**
Rephrase TLS termination.

Reported by KellerFuchs on IRC (thanks!)

### CODE REVIEW

**Comment by gdestuynder** [comment]
Technically, yes, these cases may be allowed - the intent is to.. well, carry intent.
Basically, you *should* (but not *must*) use end to end encryption where possible.
In reality it's not always possible, and what "end" means is up to interpretation.

Example:
-Web application X is a cluster of python processes serving webpages.
- Nginx runs in front of X
- Nginx terminates TLS on behalf of X

That would be an acceptable behavior even thus technically, Nginx is probably on a different machine than "X" and thus there is *some* clear-text traffic.

To make this a strong ("*must*") requirement that X must terminate TLS "in-process", we probably need a few more years - even thus we'll recommend doing so when this makes sense today (concise version of this is "please do end-to-end encryption").

If you have a better way to phrase that out though I would happily replace the sentence we currently have! These things are hard to convey in a clear and concise manner sometimes.

**Comment by gdestuynder** [comment]
@KellerFuchs  how about this version? 
https://github.com/mozilla/wikimo_content/pull/97/files#diff-599d9066ff8c736722269305fb786f56R86

File: Security/Fundamentals/Security_Principles.mediawiki [hunk 1]
```diff
@@ -83,10 +83,11 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
 * Encrypt data at rest when not actively used by the system.
 * Use full-disk encryption where available.
 * Encrypt credentials storage databases (Ansible Vault, CredStash, etc.)
-* Encrypt traffic in transit (TLS).
+* Encrypt traffic in transit (TLS) from the client to the final destination ([https://en.wikipedia.org/wiki/End-to-end_encryption end-to-end]).
+* Encrypt traffic in transit (TLS) between components of Mozilla's infrastructure when [https://en.wikipedia.org/wiki/End-to-end_encryption end-to-end] encryption is not technically feasible.
```
Comment (hwine) [inline Security/Fundamentals/Security_Principles.mediawiki]: I think this conflicts with the line above. Do you mean "Encrypt traffic between the client and app's network edge if end-to-end is not feasible"? (Allows plain text from LB to app.)

File: Security/Fundamentals/Security_Principles.mediawiki [hunk 2]
```diff
@@ -83,10 +83,11 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
...
 
-''Do not'''
-* Terminate TLS for all services in one location, then transmit the information in clear-text in the rest of the network.
+'''Do not'''
+* Terminate TLS for all services in a single location at the network edge and transmit the information in clear-text in the rest of the network.
```
Comment (hwine) [inline Security/Fundamentals/Security_Principles.mediawiki]: This sounds like "don't use LB's". Does "single location" mean "VIP", hardware?

**Comment by gdestuynder** [comment]
This was fixed by commit 2cc881acff8fce655a69b38256ebf064a22a7cee i another PR
feel free to open an issue though if there are concerns with the new wording

thanks!

**Pull request closed without merging**